addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -431,6 +431,91 @@ QUnit.test('show message subject if thread is mailing channel', async function (
     );
 });
 
+QUnit.test('message list for tracking field', async function (assert) {
+    assert.expect(3);
+    await this.start();
+    this.data["mail.message"].fields.tracking_value_ids = {
+        relation: "mail.tracking.values",
+        string: "Tracking Values",
+        type: "many2many",
+    }
```

Comment:
Add missing field in `mock_models` file.

This is the result of a formatter though, so it's not a related field in JS. It's just an array of objects with a specific shape.